tests/unit/logs/test_transformation_full.py

```diff
@@ -503,6 +503,72 @@
             'audit.result': 'Succeeded'}
     }, id="testcase_Cloudtrail_RDS_CreateEventSubscription_multiple_resources_with_source_type"),
 
+    pytest.param({
+        "record_data_decoded": {
+            "logGroup": "/aws/rds/instance/mysql-db-for-logs/audit",
+            "logStream": "mysql-db-for-logs",
+            "messageType": "DATA_MESSAGE",
+            "owner": "444000444",
+            "subscriptionFilters": ["mysql-audit-filter"],
+            "logEvents": [
+                {
+                    "id": "35958590510527767165636549608812769529777864588249006080",
+                    "timestamp": "12345",
+                    "message": "20210722 17:15:28,ip-10-1-3-230,admin,157.25.19.100,13,0,DISCONNECT,testdb,,0,SSL"
```

Comment:
Just to make sure as we looked at this code earlier - this `testdb` here (that sometimes is present, sometimes isn't) is not the just some internal mysql database name that we don't need. And the actual AWS instance name in which we're interested in to have a link to entity is already there in logGroup.

Reply:
Couldn't understand this comment at first. This one superfluous `not` is super confusing, especially that the rest of the sentence has entirely different meaning then :)

Reply:
Maria confirmed, resolving.